src/app/shared/admin/admin-candidates/admin-elections-candidate-modal.component.ts

```diff
@@ -18,6 +20,7 @@ export class AdminElectionsCandidateModalComponent implements OnInit {
   @Input() election_type: string;
   @Input() candidates: Candidate[];
   @Input() positions: Position[];
+  notSaved: boolean;
```

Comment:
lets set this to false in the constructor so it's never undefined. A different name might be better, since it's only used when adding a new candidate.